.travis.yaml

```diff
@@ -0,0 +1,52 @@
+language: android
+jdk: oraclejdk8
+
+env:
+  global:
+    - ANDROID_TARGET=android-22
```

Comment:
should this be higher then 22?

Reply:
Gonna increase to the max supported: 29.